packages/@okta/vuepress-site/docs/reference/api/trusted-origins/index.md

```diff
@@ -548,7 +812,110 @@ curl -X PUT \
 }
 ```
 
+#### Request example with iFrame embedding
+
+<ApiLifecycle access="ea" />
+
+```bash
+curl -X PUT \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+-d '{
+  "id": "tosue7JvguwJ7U6kz0g3",
+  "name": "Updated Example Trusted Origin",
+  "origin": "http://updated.example.com",
+  "scopes": [
+    {
```

Comment:
Again, are the CORS and REDIRECT scopes required in this request.

Reply:
I have updated the example here too but i have some example which shows the combination of various scope types in the request body.